<?xml version="1.0"?>
<launch>
    <arg name="minSpeed" default="-1.2"/>
    <arg name="maxSpeed" default="1.2"/>
    <arg name="wheelBase" default="0.2"/>

    <node name="limo_motion_controller" pkg="limo_motion_controller" type="limo_motion_controller" output="screen">
        <param name="minSpeed" value="$(arg minSpeed)"/>
        <param name="maxSpeed" value="$(arg maxSpeed)"/>
        <param name="wheelBase" value="$(arg wheelBase)"/>
    </node>
</launch>